rollbar-java/src/main/java/com/rollbar/notifier/sender/queue/DiskQueue.java

```diff
@@ -139,6 +145,13 @@ private static Payload read(File file, boolean remove) {
       }
       return (Payload) o;
     } catch (Exception e) {
+      if(objectInput != null) {
+        try {
+          objectInput.close();
+        } catch (IOException e1) {
+          e1.printStackTrace();
```

Comment:
I think it'd be better to use a Logger and log this error instead of calling the `printStackTrace` method.

Reply:
Ok.